vNext/Build.cmd

```diff
@@ -52,9 +38,13 @@ call %_dotnet% build %BUILD_FLAGS% /p:ArtifactDirectory=%OutputPath%\;Configurat
 @if ERRORLEVEL 1 GOTO :ErrorStop
 @echo BUILD ok for %CONFIGURATION% %SOLUTION%
 
-call %_dotnet% pack %PACK_FLAGS% /p:ArtifactDirectory=%OutputPath%\;Configuration=%CONFIGURATION% "%SOLUTION%"
+call %_dotnet% restore "%CodeGenProject%"
+@if ERRORLEVEL 1 GOTO :ErrorStop
+@echo BUILD ok for %CONFIGURATION% %CodeGenProject%
```

Comment:
Noone reads these :-)

I don't see this as an issue, but if you insist I can update it.